pkg/domain/infra/abi/play.go

```diff
@@ -866,23 +869,83 @@ func (ic *ContainerEngine) playKubePVC(ctx context.Context, pvcYAML *v1.Persiste
 			opts["GID"] = v
 		case util.VolumeMountOptsAnnotation:
 			opts["o"] = v
+		case util.VolumeImportSourceAnnotation:
+			importFrom = v
 		}
 	}
 	volOptions = append(volOptions, libpod.WithVolumeOptions(opts))
 
+	// Validate the file and open it before creating the volume for fast-fail
+	var tarFile *os.File
+	if importFrom != "" {
```

Comment:
optional: my preference is if len(var) < 1

Reply:
I guess you meant `len(var) > 0` right?